Expensify/App
# Fix keyboard bug when opening attachment modal

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/pull/2424#issuecomment-821737903

### Tests / QA Steps (iOS/Android)
1. Focus the main text input on a report.
1. Click on the `+`, verify that the create menu opens.
1. Close the modal, verify that the text input regains focus.
1. Click on the `+`, and select `Add Attachment`
1. Verify that the `CreateMenu` modal closes and is replaced by the `AttachmentPickerMenu`
1. Close the modal, verify that the text input regains focus.

### Tests / QA Steps (web/desktop/mWeb)
Just perform regression testing of the report action compose and attachment modals.

### Tested On


### Screenshots

#### Web
n/a

#### Mobile Web
n/a

#### Desktop
n/a

#### iOS
https://user-images.githubusercontent.com/47436092/115286731-3ec57080-a104-11eb-9315-117df1eef42d.mov

#### Android
https://user-images.githubusercontent.com/47436092/115293181-dbd7d780-a10b-11eb-93ec-99dd957dbdba.mov
